jhub_apps/service/app_from_git.py

```diff
@@ -0,0 +1,126 @@
+import os
+import pathlib
+import tempfile
+from pathlib import Path
+
+import git
+from fastapi import HTTPException
+from pydantic import ValidationError
+from starlette import status
```

Comment:
This is partially a matter of preference and 

```suggestion
from fastapi import HTTPException, status
from pydantic import ValidationError
```